### ISSUE TITLE
404 GET /kernelspecs/julia-0.3/logo-64x64.png on Windows

### ISSUE THREAD

**Issue by n-s-k** [JuliaLang/IJulia.jl#366]
```
jupyter notebook
[I 19:42:18.974 NotebookApp] Serving notebooks from local directory: C:\Users\###
[I 19:42:18.974 NotebookApp] 0 active kernels
[I 19:42:18.974 NotebookApp] The IPython Notebook is running at: http://localhost:8888/
[I 19:42:18.974 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[...]
[I 19:42:22.653 NotebookApp] Creating new notebook in
[I 19:42:23.105 NotebookApp] Kernel started: ####
[W 19:42:23.114 NotebookApp] 404 GET /kernelspecs/julia-0.3/logo-64x64.png (::1) 8.00ms referer=http://localhost:8888/notebooks/Untitled.ipynb?kernel_name=julia-0.3
Starting kernel event loops.
```

and the nice IJulia logo is not displayed.

**Comment by stevengj** [comment]
Ah, same thing on my machine.  I hadn't even noticed.

**Comment by yuyichao** [comment]
I've been seeing this since the first time I installed IJulia but I thought that is just because of my system wise IJulia installation is somewhat special........

**Comment by stevengj** [comment]
@minrk, are we doing something wrong?  I seem to have the logo file in my kernelspec directory:
```
MACSTATION004:julia stevenj$ ls /Users/stevenj/Library/Jupyter/kernels/julia-0.5
kernel.json	logo-32x32.png	logo-64x64.png
```
and yet I'm getting a 404 error:
```
[I 17:30:02.262 NotebookApp] Creating new notebook in 
[I 17:30:02.852 NotebookApp] Kernel started: 6a38fd68-a548-4ae7-b39d-c874f06010e0
[W 17:30:02.860 NotebookApp] 404 GET /kernelspecs/julia-0.5/logo-64x64.png (::1) 3.32ms referer=http://localhost:8888/notebooks/Untitled.ipynb?kernel_name=julia-0.5
Starting kernel event loops.
```
This is with Jupyter 4.0.4.

**Comment by minrk** [comment]
@stevengj nope, just a bad regex on our side. Fixed by jupyter/notebook#519. Sorry about that.

**Comment by stevengj** [comment]
No problem, thanks for the quick fix!
